docs/writing-tests/test-runner.md

```diff
@@ -238,6 +237,105 @@ When the test-runner executes, your existing tests will go through the following
 - The story is rendered, and any existing `play` functions are executed.
 - The `postRender` function is executed.
 
+### (Experimental) Filter tests
+
+When you run the test-runner on Storybook, it tests every story by default. However, if you want to filter the tests, you can use the `tags` configuration option. Storybook originally introduced this feature to generate [automatic documentation](../writing-docs/autodocs.md) for stories. But it can be further extended to configure the test-runner to run tests according to the provided tags using a similar configuration option or via CLI flags (e.g., `--includeTags`, `--excludeTags`, `--skipTags`). Listed below are the available options and an overview of how to use them.
```

Comment:
Do you think we need to mention that this is available from version 0.15.0 onwards? Or it doesn't matter